pom.xml

```diff
@@ -110,4 +113,14 @@
     <!--</plugins>-->
   </build>
 
+  <profiles>
+    <profile>
+      <id>solution</id>
+      <properties>
+        <source.dir>src/main/solution</source.dir>
+        <workshop.solution>true</workshop.solution>
```

Comment:
Is this used?

Reply:
No, not really used. Leftover when porting Clement's work.